js/windowMgr.js

```diff
@@ -221,6 +221,13 @@ function doCreateMainWindow(initialUrl, initialBounds, isCustomTitleBar) {
         }
     }
 
+    // Event needed to hide native menu bar on Windows 10 as we use custom menu bar
+    mainWindow.webContents.once('did-start-loading', () => {
+        if (isWindows10() && mainWindow && !mainWindow.isDestroyed()) {
+            mainWindow.setMenuBarVisibility(false);
```

Comment:
what does this do?

Reply:
This will hide the menu bar on `Windows`